jujugui/static/gui/src/app/assets/javascripts/jujulib/juju.js

```diff
@@ -407,7 +412,7 @@ var module = module;
         }
         processed.deployerFileUrl =
             this.url +
-            this.version + '/' +
+            this.version + '/' + 
```

Comment:
Remove trailing whitespace. There is probably a setting in your editor to trim trailing whitespace on save.